x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/steps/add_fleet_server_host.tsx

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+/* eslint-disable @elastic/eui/href-or-on-click */
```

Comment:
I did the same that is done for the other button in `components/fleet_server_instructions/steps/get_started.tsx` but it can be disabled on the interested lines only. I'll fix both!